Dockerfile_mbtci_template

```diff
@@ -93,6 +93,14 @@ RUN ARCH= && dpkgArch="$(dpkg --print-architecture)" \
     && node --version \
     && npm --version
 
+# Install Grunt CLI
+RUN set -ex \
+    && npm install --prefix /usr/local/ -g grunt-cli \
```

Comment:
```suggestion
    && npm install -g grunt-cli \
```

Reply:
Make the install way same as UI5

Reply:
The prefix option was and is still not needed. 
The version is not enforced.